.github/actions/matches/action.yml

```diff
@@ -0,0 +1,50 @@
+name: "File Matching"
+description: |
```

Comment:
needs a readme, but pretty simple and works pretty well, after all it is just a grep.